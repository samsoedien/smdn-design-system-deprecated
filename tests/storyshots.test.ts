import initStoryshots from '@storybook/addon-storyshots' // eslint-disable-line

initStoryshots()
