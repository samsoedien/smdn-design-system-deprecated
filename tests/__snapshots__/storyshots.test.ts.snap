// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Charts/Chart Chart - Default 1`] = `
<canvas
  className="sc-AxhCb ccMvAG smdn-chart"
  data-test="chart-component"
  type="lineseries"
/>
`;

exports[`Storyshots Charts/Metric Metric - Default 1`] = `
<div
  className="sc-AxhUy kMyssT smdn-metric"
  data-test="metric-component"
>
  <span>
    --
  </span>
</div>
`;

exports[`Storyshots Charts/Sparkline Sparkline - Default 1`] = `
<div
  className="sc-AxgMl bFfuHh smdn-sparkline"
  data-test="sparkline-component"
>
  <div>
    summary
  </div>
  <div
    className="sc-AxhUy kMyssT smdn-metric"
    data-test="metric-component"
  >
    <span>
      --
    </span>
  </div>
</div>
`;

exports[`Storyshots Components/Accordion Accordion - Default 1`] = `
<div
  className="sc-fzqBkg cifTAZ smdn-accordion"
  data-test="accordion-component"
>
  Accordion
</div>
`;

exports[`Storyshots Components/Avatar Avatar - Default 1`] = `
<img
  alt="John"
  className="sc-fzqPZZ ijMfzc smdn-avatar"
  data-test="avatar-component"
  type="user"
/>
`;

exports[`Storyshots Components/Badge Badge - Default 1`] = `
<span
  className="sc-fzoxKX ja-DdBK smdn-badge"
  data-test="badge-component"
/>
`;

exports[`Storyshots Components/Breadcrumbs Breadcrumbs - Default 1`] = `
<div
  className="sc-fzoKki fatNHo smdn-breadcrumbs"
  data-test="breadcrumbs-component"
/>
`;

exports[`Storyshots Components/Button Button - Default 1`] = `
<button
  className="sc-fzoYkl cXbrPw smdn-b-button"
  data-test="button-component"
  onClick={[Function]}
  size="regular"
  type="submit"
>
  Default Button
</button>
`;

exports[`Storyshots Components/Button Button - Loading 1`] = `
<button
  className="sc-fzoYkl eIDsBq smdn-b-button"
  data-test="button-component"
  onClick={[Function]}
  size="regular"
  type="submit"
>
  Loading...
</button>
`;

exports[`Storyshots Components/Button Button - Primary 1`] = `
<button
  className="sc-fzoYkl UkEVb smdn-b-button"
  data-test="button-component"
  onClick={[Function]}
  size="regular"
  type="submit"
>
  Primary Button
</button>
`;

exports[`Storyshots Components/Button Button - Secondary 1`] = `
<button
  className="sc-fzoYkl cXbrPw smdn-b-button"
  data-test="button-component"
  onClick={[Function]}
  size="regular"
  type="submit"
>
  Secondary Button
</button>
`;

exports[`Storyshots Components/Button Button - Sizes 1`] = `
Array [
  <button
    className="sc-fzoYkl cCFBxD smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="small"
    type="submit"
  >
    Small
  </button>,
  <button
    className="sc-fzoYkl cXbrPw smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="regular"
    type="submit"
  >
    Regular
  </button>,
  <button
    className="sc-fzoYkl eJdNYC smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="large"
    type="submit"
  >
    Large
  </button>,
]
`;

exports[`Storyshots Components/Button Button - Tertiary 1`] = `
<button
  className="sc-fzoYkl dwrsoK smdn-b-button"
  data-test="button-component"
  onClick={[Function]}
  size="regular"
  type="submit"
>
  Tertiary Button
</button>
`;

exports[`Storyshots Components/Button Buttons 1`] = `
Array [
  <button
    className="sc-fzoYkl UkEVb smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="regular"
    type="submit"
  >
    Primary
  </button>,
  <button
    className="sc-fzoYkl cXbrPw smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="regular"
    type="submit"
  >
    Secondary
  </button>,
  <button
    className="sc-fzoYkl dwrsoK smdn-b-button"
    data-test="button-component"
    onClick={[Function]}
    size="regular"
    type="submit"
  >
    Tertiary
  </button>,
]
`;

exports[`Storyshots Components/Callout Callout - Default 1`] = `
<div
  className="sc-fzpkJw hwAyNc smdn-callout"
  color="info"
  data-test="callout-component"
>
  Hello world
</div>
`;

exports[`Storyshots Components/Card Card - Default 1`] = `
<div
  className="sc-fznzOf gEmGWT smdn-card"
  color="white"
  data-test="card-component"
  elevation={1}
>
  Hello World
</div>
`;

exports[`Storyshots Components/CardContent Card Content - Default 1`] = `
<section
  className="sc-fznMnq jaPXIn smdn-card-content"
  data-test="card-content-component"
>
  Hello World
</section>
`;

exports[`Storyshots Components/CardFooter Card Footer - Default 1`] = `
<footer
  className="sc-fzoant gHvbhb smdn-card-footer"
  data-test="card-footer-component"
>
  Hello World
</footer>
`;

exports[`Storyshots Components/CardHeader Card Header - Default 1`] = `
<header
  className="sc-fzomME kwpkdO smdn-card-header"
  data-test="card-header-component"
>
  Hello World
</header>
`;

exports[`Storyshots Components/Code Code - Default 1`] = `
<code
  className="sc-fzqAbL zYCzw smdn-code"
  data-test="code-component"
>
  Code
</code>
`;

exports[`Storyshots Components/Combobox Combobox - Default 1`] = `
<div
  className="sc-fzoYHE jFJDaF smdn-combobox"
  data-test="combobox-component"
>
  <div
    className="sc-fzoydu kUYJSZ smdn-combobox-input"
    data-test="combobox-input-component"
  >
    <span
      className="sc-fzqMAW fcsxmK smdn-tag"
      color="#efefef"
      data-test="tag-component"
    >
      <span>
        filter
      </span>
    </span>
  </div>
  <div
    className="sc-fzoJMP gAzsmC smdn-combobox-option-list"
    data-test="combobox-option-list-component"
  />
</div>
`;

exports[`Storyshots Components/Comment Comment - Default 1`] = `
<div
  className="sc-fzpkqZ kmZxWu smdn-comment"
  data-test="comment-component"
>
  Comment
</div>
`;

exports[`Storyshots Components/Dialog Dialog - Default 1`] = `
<div
  className="sc-fznAgC dbXiMN smdn-dialog"
  data-test="dialog-component"
>
  Dialog
</div>
`;

exports[`Storyshots Components/Divider Divider - Default 1`] = `
<hr
  className="sc-fznLPX jCbNnQ smdn-divider"
  data-test="divider-component"
/>
`;

exports[`Storyshots Components/Input Input - Default 1`] = `
<input
  className="sc-fzoaKM iONrUQ smdn-input"
  data-test="input-component"
  name="name"
  onChange={[Function]}
  placeholder="Type Something..."
  type="text"
/>
`;

exports[`Storyshots Components/ListGroup ListGroup - Default 1`] = `
<ul
  className="sc-fzomuh eGtsA-d smdn-list-group"
  data-test="list-group-component"
>
  List Group
</ul>
`;

exports[`Storyshots Components/Spacer Spacer - Default 1`] = `
<br
  className="sc-fzqAui jAAClJ smdn-spacer"
  data-test="spacer-component"
  size="sm"
/>
`;

exports[`Storyshots Components/Table Table - Default 1`] = `
<table
  className="sc-fzqMdD haaTSx smdn-table"
  data-test="table-component"
>
  <thead
    className="sc-fzowVh gSfeHE smnd-table-heade"
    data-test="table-head-component"
  >
    <tr
      className="sc-fzoJus cXGpQF smdn-table-row"
      data-test="table-row-component"
    >
      <th
        className="sc-fzoVTD fwKNiT smdn-table-cell"
        data-test="table-cell-component"
      >
        Heading 1
      </th>
      <th
        className="sc-fzoVTD fwKNiT smdn-table-cell"
        data-test="table-cell-component"
      >
        Heading 2
      </th>
    </tr>
  </thead>
  <tbody
    className="sc-fznyYp ePFTkY smdn-table-body"
    data-test="table-body-component"
  >
    <tr
      className="sc-fzoJus cXGpQF smdn-table-row"
      data-test="table-row-component"
    >
      <td
        className="sc-fzpisO ciFKdv smdn-table-cell"
        data-test="table-cell-component"
      >
        Body 1
      </td>
      <td
        className="sc-fzpisO ciFKdv smdn-table-cell"
        data-test="table-cell-component"
      >
        Body 2
      </td>
    </tr>
    <tr
      className="sc-fzoJus cXGpQF smdn-table-row"
      data-test="table-row-component"
    >
      <td
        className="sc-fzpisO ciFKdv smdn-table-cell"
        data-test="table-cell-component"
      >
        Body 3
      </td>
      <td
        className="sc-fzpisO ciFKdv smdn-table-cell"
        data-test="table-cell-component"
      >
        Body 4
      </td>
    </tr>
  </tbody>
</table>
`;

exports[`Storyshots Components/Tabs Tabs - Default 1`] = `
<div
  className="sc-fznLxA kCDRyF smdn-tabs"
  data-test="tabs-component"
>
  Tabs
</div>
`;

exports[`Storyshots Components/Tag Tag - Default 1`] = `
<span
  className="sc-fzqMAW fcsxmK smdn-tag"
  color="#efefef"
  data-test="tag-component"
>
  <span>
    Hello World
  </span>
</span>
`;

exports[`Storyshots Components/Textarea Textarea - Default 1`] = `
<textarea
  autofocus={false}
  className="sc-fznXWL fxhNWV smdn-textarea"
  cols={50}
  data-test="textarea-component"
  disabled={false}
  name="name"
  onChange={[Function]}
  rows={4}
>
  Hello world
</textarea>
`;

exports[`Storyshots Components/Toast Toast - Default 1`] = `
<div
  className="sc-fzokvW kJbQFf smdn-toast"
  color="info"
  data-test="toast-component"
>
  Hello world
</div>
`;

exports[`Storyshots Components/Tooltip Tooltip - Default 1`] = `
<div
  className="sc-fzqzlV eoCMvb smdn-tooltip"
  data-test="tooltip-component"
>
  Hello world
</div>
`;

exports[`Storyshots Layouts/HeroBanner HeroBanner - Default 1`] = `
<div
  className="sc-fznzqM iQOVNS smdn-hero-banner"
  data-test="hero-banner-component"
>
  Hello world
</div>
`;

exports[`Storyshots Layouts/Page Page - Default 1`] = `
<div
  className="sc-fznOgF kUlBsE smdn-page"
  data-test="page-component"
>
  Hello world
</div>
`;

exports[`Storyshots Layouts/PageContent PageContent - Default 1`] = `
<div
  className="sc-fznYue djsYXx smdn-page-content"
  data-test="page-content-component"
>
  Hello world
</div>
`;

exports[`Storyshots Layouts/PageFooter PageFooter - Default 1`] = `
<footer
  className="sc-fzonjX dzycXD smdn-page-footer"
  data-test="page-footer-component"
>
  Hello world
</footer>
`;

exports[`Storyshots Layouts/PageHeader PageHeader - Default 1`] = `
<header
  className="sc-fzqzEs ejSgor smdn-page-header"
  data-test="page-header-component"
>
  Hello world
</header>
`;

exports[`Storyshots Layouts/PageSideBar PageSideBar - Default 1`] = `
<div
  className="sc-fzqOul JAzeZ smdn-page-side-bar"
  data-test="page-side-bar-component"
>
  Hello world
</div>
`;
